build.sbt

```diff
@@ -217,3 +219,16 @@ lazy val scalametaSettings = Seq(
   scalacOptions += "-Xplugin-require:macroparadise",
   publishArtifact in (Compile, packageDoc) := false
 )
+
+coverageExcludedFiles in Global  := ".*<macro>"
+
+def toCompileTestList(sequence: Seq[ProjectReference]): List[String] =
+  sequence.toList.map(p => s"${p.asInstanceOf[LocalProject].project}/test")
```

Comment:
casting inside a StringContext?, we are going to hell for this but whatever! :shipit: